auth/protocol.go

```diff
@@ -91,12 +91,12 @@ type AuthorizationRequest struct {
 }
 
 const  (
-	// METADATA_INTERAL signifies that a value is internal and for use only in containerssh
-	METADATA_INTERNAL     = 0
-	// METADATA_CONFIG signifies that a value can be exposed to the configuration server
-	METADATA_CONFIG       = 1 << 0
-	// METADATA_CONTAINER signifies that a value can be exposed to the pod as environment variables or files
-	METADATA_CONTAINER    = 1 << 1
+	// MetadataInternal signifies that a value is internal and for use only in containerssh
+	MetadataInternal     = 0
+	// MetadataConfig signifies that a value can be exposed to the configuration server
+	MetadataConfig       = 1 << 0
```

Comment:
It would be better to use `iota` here.

Reply:
This was removed in `077623b`, did you review an older revision?

Reply:
Possibly, I went through the commits.